migrations/generic/timetables/1676027954829_add_hastus_fields/up.sql

```diff
@@ -0,0 +1,52 @@
+CREATE TABLE vehicle_journey.journey_type (
+  type text PRIMARY KEY
+);
+COMMENT ON TABLE vehicle_journey.journey_type IS 'Enum table for defining allowed journey types.';
+
+INSERT INTO vehicle_journey.journey_type
+  (type)
+  VALUES
+  ('STANDARD'),
+  ('DRY_RUN'),
+  ('SERVICE_JOURNEY')
+  ON CONFLICT (type) DO NOTHING;
+
+ALTER TABLE vehicle_journey.vehicle_journey
+  ADD COLUMN journey_name_i18n jsonb,
+  ADD COLUMN turnaround_time interval,
+  ADD COLUMN layover_time interval,
+  ADD COLUMN journey_type text NOT NULL DEFAULT 'STANDARD' REFERENCES vehicle_journey.journey_type (type);
+COMMENT ON COLUMN vehicle_journey.vehicle_journey.journey_name_i18n IS 'Name that user can give to the vehicle journey.';
+COMMENT ON COLUMN vehicle_journey.vehicle_journey.turnaround_time IS 'Turnaround time. Means the time it takes to move vehicle to the starting stop and pick up the passengers.';
+COMMENT ON COLUMN vehicle_journey.vehicle_journey.layover_time IS 'Layover time. 95% of the journeys should arrive to the terminal within journey time + recovery time so that they would be ready to start next journey in time.';
+COMMENT ON COLUMN vehicle_journey.vehicle_journey.journey_type IS 'STANDARD | DRY_RUN | SERVICE_JOURNEY';
+
+CREATE SCHEMA reusable_components;
+
+CREATE TABLE reusable_components.vehicle_type (
```

Comment:
In Transmodel, VEHICLE TYPE belongs to the Vehicle Type model. I'd rather name the schema accordingly: `vehicle_type.vehicle_type`